Expensify/react-native-live-markdown
# Implement code and pre blocks support on web

### Details

Adds support for displaying and customising code and pre blocks on web.

### Related Issues

GH_LINK

### Manual Tests

1. Change regex for `codeFence` in ExpensiMark to /(&#x60;&#x60;&#x60;(?:\r\n|\n))((?:\s*?(?!(?:\r\n|\n)?&#x60;&#x60;&#x60;(?!`))[\S])+\s*?(?:\r\n|\n))(&#x60;&#x60;&#x60;)/g
2. Run `npm run build` in `/parser`
3. You should now be able to test it out 🙌🏻 

### Linked PRs
